reri_err_reporting.adoc

```diff
@@ -77,17 +77,16 @@ range from 0 to 62.
 |  0          |`vendor_n_imp_id`  |8   |Vendor and implementation ID.
 |  8          |`bank_info`        |8   |Error bank information.
 | 16          |`valid_summary`    |8   |Summary of valid error records.
-| 24          | Reserved          |16  |Reserved for future standard use.
-| 40          | Custom            |24  |Designated for custom use.
+| 24          | Reserved          |32  |Reserved for future standard use.
+| 56          | Custom            |8   |Designated for custom use.
 | 64 + 64 * i |`control_i`        |8   |Control register of error record i.
 | 72 + 64 * i |`status_i`         |8   |Status register of error record i.
-| 80 + 64 * i |`addr_i`           |8   |Address register of error record i.
+| 80 + 64 * i |`addr_info_i`      |8   |Address-or-info. register of error record i.
```

Comment:
"Address-or-info". If it's not address, what information does it contain? Would status.at will still be one?